test/cli/test-generate/multiple-test-cases/policy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: multiple-test-cases
+  namespace: hello-world
+spec:
+  rules:
+  - name: create-default-pdb
+    match:
```

Comment:
Please add an `any` or `all` block here to be conformant with the current, modern policy language. Reference [here](https://kyverno.io/docs/writing-policies/match-exclude/).

Reply:
Thanks, addressed in 922eae48edad4b23d1a16a58738a6383b13e6585.